engine/dumps/bank03.asm

```diff
@@ -2285,30 +2285,30 @@ Datae236:
 	dw Texte23e
 
 Texte23e:
-	db "#の　ようすをみる@"
+	db "#の　ようすをみる@" ; (lit "look at Pokemon")
 
 Texte248:
```

Comment:
```suggestion
WithdrawMonText:
```

Eventually these labels might evolve to include "BillsPC" or something to that effect... but until we confirmed they are related.. just keep it simple for now.